back/app/models/initiative.rb

```diff
@@ -96,9 +96,8 @@ class Initiative < ApplicationRecord
   end)
 
   scope :with_status_code, (proc do |code|
-    joins('LEFT OUTER JOIN initiative_initiative_statuses ON initiatives.id = initiative_initiative_statuses.initiative_id')
-      .joins('LEFT OUTER JOIN initiative_statuses ON initiative_statuses.id = initiative_initiative_statuses.initiative_status_id')
-      .where(initiative_statuses: { code: code })
+    left_outer_joins(:initiative_initiative_status)
```

Comment:
Sorry for so many comments about these lines, but I've just realized that it can be as simple as
```suggestion
     Initiative.joins(:initiative_status).where(initiative_status: { code: code} )
```
First, it was introduced here https://github.com/CitizenLabDotCo/citizenlab/commit/a162b35 and I'm not sure why it was so complicated.

Reply:
No problem. Nice find!

Might it be that the SQL version is (slightly) faster?

Reply:
Yes, I mean the process of generating the SQL takes some time (very little)